style_editor.html

```diff
@@ -1046,12 +1046,14 @@
 var width;
 var height;
 
-// Create n textures of about 1MB each.
-function initGL() { 
+function initGL() {
   AddTab("color", "Styles",effect_links.sort().join(""))
+  // Sort sortedRgbLinks based on the name:
+  sortedRgbLinks.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
I think colors should be sorted by their position on the color wheel rather than by name actually.

Reply:
Could be cool to populate a pie chart with slices for buttons. Beyond my scope at the moment though.

Reply:
But for now, alphabetical makes more sense than random to me. If I'm looking for a color, it's certainly easier to find.

Reply:
But they are almost sorted by color already...

Reply:
Totally up to you. Want to take a vote?

Reply:
![Screenshot 2023-06-02 at 9 04 31 AM](https://github.com/profezzorn/ProffieOS-StyleEditor/assets/53964195/bd0e7699-9d50-4db9-b65d-673daedf401b)